docs/central-upgrade.rst

```diff
@@ -10,6 +10,7 @@ Start by reviewing upgrade notes for all versions between your current version a
 Upgrade notes
 -------------
 
+* :ref:`Central v2024.2 <central-upgrade-2024.2>`: configuration no longer requires rebuild, consider using S3 storage
```

Comment:
Might be worth updating this line to match the recent changes to below?

```suggestion
* :ref:`Central v2024.2 <central-upgrade-2024.2>`: some configuration no longer requires rebuild, consider using S3 storage
```

Reply:
I matched the section title, hopefully that seems ok.